.github/workflows/SPDX-README.md

```diff
@@ -0,0 +1,31 @@
+Pleasing checkpatch is hard when adding new files because:
+
+- checkpatch wants a different SPDX style for .c versus .h files!
+- SOF rejects C99 comments starting with //
+
+The trick is to keep the SPDX separate. See solution below.
+
+References:
+- https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/process/license-rules.rst#n71
+- https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=9f3a89926d6d
+
+
+Start .h files like this:
+```
+/* SPDX-License-Identifier: ... */
```

Comment:
is there a way we could unify this or is this a silly standard?

Reply:
The latter I'm afraid.

As I mentioned a while back in https://github.com/thesofproject/sof/pull/6284#issuecomment-1252739170, this .h versus .c file is "by design", see Linux commit 9f3a89926d6d